lib/core/src/Cardano/Wallet/DB/Sqlite.hs

```diff
@@ -432,6 +433,29 @@ newDBLayer logConfig trace mDatabaseFile = do
         , readPrivateKey = \(PrimaryKey wid) ->
               runQuery $ selectPrivateKey wid
 
+        {-----------------------------------------------------------------------
+                                       Pending Tx
+        -----------------------------------------------------------------------}
+
+        , removePendingTx = \(PrimaryKey wid) tid ->
+              ExceptT $ runQuery $
+              selectWallet wid >>= \case
+                  Just _ -> do
+                      metas <- selectPendingTxs wid (TxId tid)
+                      let checkStatus = not . any
+                              (\(TxMeta _ _ st _ _ _ _) -> st == W.Pending)
```

Comment:
I'd suggest to use `isPending` from the `Primitive.Types` :+1:

Reply:
indeed, corrected